pkgs/development/compilers/elm/packages/elm-format.nix

```diff
@@ -1,24 +1,28 @@
 { mkDerivation, fetchgit, ansi-terminal, ansi-wl-pprint, base, binary
-, bytestring, Cabal, cmark, containers, directory, filepath, free
-, HUnit, indents, json, mtl, optparse-applicative, parsec, process
+, bytestring, cmark, containers, directory, filepath, free, HUnit
+, indents, json, mtl, optparse-applicative, parsec, process
 , QuickCheck, quickcheck-io, split, stdenv, tasty, tasty-golden
 , tasty-hunit, tasty-quickcheck, text
 }:
 mkDerivation {
   pname = "elm-format";
-  version = "0.8.1";
+  version = "0.8.2";
   src = fetchgit {
     url = "https://github.com/avh4/elm-format";
-    sha256 = "0p1dy1m6illsl7i04zsv5jqw7i4znv7pfpdfm53zy0k7mq0fk09j";
-    rev = "89694e858664329e3cbdaeb71b15c4456fd739ff";
+    sha256 = "0ly37fszrfviwqgrww57ikdcr7i8lcpczhqm8xqp5s7mrlpdxv7z";
+    rev = "ab3627cce01e5556b3fe8c2b5e3d92b80bfc74af";
   };
   postPatch = ''
-    sed -i "s|desc <-.*||" ./Setup.hs
-    sed -i "s|gitDescribe = .*|gitDescribe = \\\\\"0.8.1\\\\\"\"|" ./Setup.hs
+    mkdir -p ./generated
+    cat <<EOHS > ./generated/Build_elm_format.hs
+    module Build_elm_format where
+
+    gitDescribe :: String
+    gitDescribe = "0.8.2"
+    EOHS
   '';
-  isLibrary = true;
+  isLibrary = false;
```

Comment:
The nix file that includes this one uses `justStaticExecutables` (I think because building just the library would fail on MacOS?), so it was suggested that the library shouldn't be exposed.  I think this is the appropriate change?